sdk/keyvault/azure-keyvault-certificates-apis/azure/keyvault/certificates/enums.pyi

```diff
@@ -0,0 +1,37 @@
+# Stubs for azure.keyvault.certificates.enums (Python 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from enum import Enum
+
+class ActionType(str, Enum):
```

Comment:
We don't expose this in .NET, but we do it differently. If you need to expose this, it should be CertificateActionType to coincide with CertificateAction.